src/generate.rs

```diff
@@ -435,6 +486,9 @@ when_full = "block"
 inputs = []
 type = "add_fields"
 
+[transforms.transform0.fields]
+name = "TODO"
```

Comment:
I this still TODO?

Reply:
This is generated from fields.name = "TODO" above.